docs/reference/commandline/build.md

```diff
@@ -29,7 +29,7 @@ parent = "smn_cli"
       -m, --memory=""                 Memory limit for all build containers
       --memory-swap=""                A positive integer equal to memory plus swap. Specify -1 to enable unlimited swap.
       --no-cache                      Do not use cache when building the image
-      --pull                          Always attempt to pull a newer version of the image
+      --pull=<content-trust>          Always attempt to pull a newer version of the image
```

Comment:
*Does this mean, <content-trust> is a special value that's the default value?*

`<content-trust>` is a special value that is the equivalent to the results of `isTrusted()`

 *But if you specify --pull then it always pulls?* 

No, if you specify `--pull` (or `--pull=true`), pull behavior will be set to either `PullMissing` or `PullAlways`.  Additionally, the translator function may be set to `trustedReference()`.  See the table in the PR description.

*And --pull=false never?*

No, if you specify `--pull=false`, pull behavior will be set to either `PullMissing` or `PullNever`.  In both cases the translator will not be set to `trustedReference()`.  See the table in the PR description.